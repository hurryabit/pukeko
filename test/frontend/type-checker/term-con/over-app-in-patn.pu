data T a = | C a
f :: T a -> a
f t = case t of
  | C x y -> x
