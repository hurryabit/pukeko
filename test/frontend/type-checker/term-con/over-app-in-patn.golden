test/frontend/type-checker/term-con/over-app-in-patn.pu:3:7: data constructor 'C' expects 1 arguments
